Releasenotes.md

```diff
@@ -1,6 +1,76 @@
 # Release notes BRK-Bevragen
 
-## Versie 1.1.1:
+## **Versie 1.2.0:**
+
+### openapi.yaml
+
+- Algemeen
+  - Diverse verbeteringen in de documentatie (descriptions) en examples.
+  - Er is gebruikgemaakt van Haal-Centraal-common versie 1.2.0 (was 1.0.0)
+  - De default-waarde voor de Accept-Crs header is verwijderd.
+  - Uit alle componentnamen zijn de underscores verwijderd.
+
+
+- KadastraalOnroerendeZaak
+  - De default-waarde voor de Accept-Crs header is verwijderd bij /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}
+  - PrivaatrechtelijkeBeperkingen, hypotheken en beslagen zijn toegevoegd aan de _embedded en _links. De bijbehorende identificaties zijn opgenomen in de KadastraalOnroerendeZaak.
+  - LocatieKadastraalObject levert naast nummeraanduidingIdentificatie nu ook het adres.
```

Comment:
is het handig hier de naam van het schemacomponent te benoemen? Of beter de naam van het property in de response? (ik denk het laatste)
naast nummeraanduidingIdentificatie zat er ook al koppelingswijze in.

Reply:
Aangapast met de propertynamen. 
Ik wil duidelijk maken dat naast alleen een verwijzing naar het adres nu ook het adres zelf is opgenomen, niet zozeer opsommen wat er al in de locatieKadastraalObject zat.